R/join_abcd_scenario.R

```diff
@@ -160,11 +160,13 @@ add_green_technologies_to_abcd <- function(data, scenario) {
     unique() %>%
     inner_join(increasing_techs, by = c("sector", "technology"))
 
-  increasing_techs_not_in_abcd <- dplyr::filter(
+  increasing_techs_not_in_abcd <- dplyr::anti_join(
     increasing_techs_in_scenario,
-    !(.data[["technology"]] %in% unique(data$technology))
+    data,
+    by = c("sector", "technology")
   )
 
+  # TODO: the summarize should be replaced with a distinct. the left_join should be an inner_join to avoid adding NAs
```

Comment:
Should this TODO be converted into an issue?

Reply:
https://github.com/RMI-PACTA/r2dii.analysis/issues/497